Backend/src/main/java/com/valantic/myplaylist/TrackController.java

```diff
@@ -19,4 +18,9 @@ public TrackController(TrackService trackService) {
     public List<Track> getTracks() {
         return trackService.getTracks();
     }
+
+    @PostMapping
+    public void addTrack(@Valid @RequestBody Track newTrack) {
```

Comment:
Wenn du willst:
Gib hier den neu hinzugefügten Track zurück

Reply:
create a method in TrackService to get the specific track by id
to get the added track (?) back from db after adding new track
TODO:  test the return value